autoemulate/experimental/emulators/lightgbm.py

```diff
@@ -0,0 +1,125 @@
+import numpy as np
+from lightgbm import LGBMRegressor
+from sklearn.base import BaseEstimator, RegressorMixin
+from sklearn.utils.validation import check_array, check_is_fitted, check_X_y
+from torch import Tensor
+
+from autoemulate.experimental.emulators.base import (
+    Emulator,
+    InputTypeMixin,
+)
+from autoemulate.experimental.types import InputLike, OutputLike
+
+
+class LightGBM(Emulator, InputTypeMixin, BaseEstimator, RegressorMixin):
```

Comment:
Is there a good reason to also inherit from the sklearn `BaseEstimator` and `RegressorMixin` classes here or should we remove these?

Reply:
Yes we should remove this. It's because in the original AutoEmulate implementation everything was made to look like an sklearn object but that is not necessary anymore.